### PR TITLE
add : array return type at ConfigProvider::__invoke()

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

File: src/ExpressiveInstaller/Resources/src/ConfigProvider.flat.php [hunk 1]
```diff
@@ -19,7 +19,7 @@ class ConfigProvider
      *
      * @return array
```
Comment (xtreamwayz) [inline src/ExpressiveInstaller/Resources/src/ConfigProvider.flat.php]: You can remove this now.

File: src/ExpressiveInstaller/Resources/src/ConfigProvider.modular.php [hunk 2]
```diff
@@ -19,7 +19,7 @@ class ConfigProvider
      *
      * @return array
```
Comment (xtreamwayz) [inline src/ExpressiveInstaller/Resources/src/ConfigProvider.modular.php]: Same.

**Comment by samsonasik** [comment]
@xtreamwayz done ;)

**Comment by xtreamwayz** [comment]
Thanx @samsonasik!

**Pull request merged**